firebase-installations/customer-lint-checks/src/main/java/com/google/firebase/installations/lint/IncompatibleIidVersionDetector.kt

```diff
@@ -59,15 +53,29 @@ class IncompatibleIidVersionDetector : Detector() {
                     if (isIncompatibleVersion(coordinates)) {
                         context.report(INCOMPATIBLE_IID_VERSION,
                                 Location.create(context.file),
-                                "Incompatible IID version found in variant ${variant.name}: ${lib.name}")
+                                "Incompatible IID version found in variant ${variant.name}: ${lib.name.removeSuffix("@aar")}.\n" +
+                                        "To resolve this issue, consider updating the build file to the latest firebase-iid version.")
                     }
                 }
             }
         }
     }
 
     private fun isIncompatibleVersion(coordinates: MavenCoordinates): Boolean {
```

Comment:
Generally it's not a good idea to use negative words in method or variable names.
Thus, I would recommend to call this #isCompatibleInstanceIdVersion